terraform/modules/hub/files/tasks/frontend.json

```
@@ -99,6 +102,9 @@
     }, {
       "Name": "AB_TEST_FILE",
       "Value": "/verify-frontend/federation/configuration/ab_test/ab_test.yml"
```

you'll want to do something like this
```suggestion
      "Value": "/verify-frontend/federation/configuration/ab_test/${ab_test_file}"
```
and then in add it here https://github.com/alphagov/verify-infrastructure-config/blob/65b5c10bc4e39436b89e182ea7f5eb512bc18c59/terraform/deployments/staging/hub/site.tf#L48

However I'd suggest doing as two separate PRs - when tearing down the AB test it will be easier to revert if the PR has only the variant stuff ;)

---

Thanks for that @jakubmiarka !